src/webots/nodes/utils/WbSupervisorUtilities.cpp

```diff
@@ -594,15 +600,21 @@ void WbSupervisorUtilities::handleMessage(QDataStream &stream) {
       stream >> nodeId;
       const QString &stateName = readString(stream);
       WbNode *const node = getProtoParameterNodeInstance(nodeId, "wb_supervisor_node_load_state()");
-      node->reset(stateName);
+      if (node)
+      {
+        node->reset(stateName);
+      }
```

Comment:
That should partially fix the clang-format issue:
```suggestion
      if (node)
        node->reset(stateName);
```